content/docs/en/elements/components/button.md

```diff
@@ -22,4 +23,13 @@ The Button component is used to display a button that can react to a tap event.
 
 | name | description |
 |------|-------------|
-| `tap`| Emitted when the button is pressed
\ No newline at end of file
+| `tap`| Emitted when the button is pressed
+
+## Native Component
+| Android | iOS |
+|---------|-----|
+| android.widget.Button | UIButton
+
+---
+contributors: [MisterBrownRSA]
```

Comment:
Same as before, this should be at the top of the file in the existing front-matter